liquidation-monitor/src/agent.spec.js

```diff
@@ -101,7 +101,7 @@ jest.mock('forta-agent', () => ({
 }));
 
 const {
-  ethers, Finding, FindingType, FindingSeverity, TransactionEvent,
+  ethers, Finding, FindingType, FindingSeverity, TransactionEvent, BlockEvent,
```

Comment:
There is a helper function in the Forta Bot SDK called `createBlockEvent` that effectively does what the function `mockBlock` does.

The main differences are that:
- `createBlockEvent` will use default values for the `type` and `network` arguments if you don't pass anything in, while `mockBlock` will always populate those with `null`
- `createBlockEvent` takes a `block` Object as an argument, whereas `mockBlock` takes a timestamp as an argument and then creates a `block` Object inside it to create the `BlockEvent` Object

You can switch to `createBlockEvent` if it seems convenient, but I can see a case for both.